### ISSUE TITLE
Compiling on archlinux.

### ISSUE THREAD

**Issue by mattaereal** [Cyb0org/DropWatch#1]
**Hi,** I wasn't able to compile it. I had to manually add the path to the libnl3 library for the netlink headers to be read.
Then there's an error while using bfd.h.
>  They claim that bfd is an "internal only" library and should not be used by other projects, and use this to justify their decision to force users to define PACKAGE and PACKAGE_VERSION themselves.

Bypassing that also results in a lot of undefined references. Have you tried to compile it recently?

Thanks!

**Comment by Cyb0org** [comment]
No. I've fork original repository for some reason a long ago. I think you will have a better help from original author (mail in readme).
I'm not sure if I've even ever used this. ;)
